src/Symfony/Component/PropertyInfo/Util/PhpDocTypeHelper.php

```diff
@@ -109,19 +111,30 @@ private function createType(DocType $type, bool $nullable, string $docType = nul
         }
 
         if ('[]' === substr($docType, -2)) {
-            if ('mixed[]' === $docType) {
-                $collectionKeyType = null;
```

Comment:
this code was never executed. in case of a `mixed[]` type on the variable, `$docType` was array, so I removed this code.